pkgs/tools/misc/atuin/default.nix

```
@@ -27,5 +27,6 @@ rustPlatform.buildRustPackage rec {
     homepage = "https://github.com/ellie/atuin";
     license = licenses.mit;
     maintainers = [ maintainers.onsails ];
+    badPlatforms = [ "aarch64-darwin" ];
```

```suggestion
    broken = [ "aarch64-darwin" ];
```

What is the error you get?

---

https://github.com/ellie/atuin/issues/220#issue-1071354928

---

I would guess that libsodium-sys does not support darwin-arach64.